examples/quickstart_huggingface/requirements.txt

```diff
@@ -1,6 +1,6 @@
-datasets~=2.12.0
-evaluate~=0.4.0
-flwr~=1.4.0
-numpy~=1.21.1
-torch~=2.0.1
-transformers~=4.29.2
+flwr>=1.0, <2.0
+torch==1.13.1
+transformers==4.11.3
+datasets==2.0.0
```

Comment:
Is our code example compatible with the new major version?

Reply:
I run the example after changing the version and it works, using datasets 1.12 is incompatible with tokenizers.

Reply:
Sounds good.